src/express-tooling.ts

```diff
@@ -103,9 +102,17 @@ function handleOAuthRequestMiddleware(options: MiddlewareOptions): RequestHandle
       return next();
     }
 
-    const accessToken = extractAccessToken(getHeaderValue(req, AUTHORIZATION_HEADER_FIELD_NAME));
+    const authHeader = getHeaderValue(req, AUTHORIZATION_HEADER_FIELD_NAME);
+
+    if (!authHeader) {
```

Comment:
mh ok thinking about it that is maybe not a good idea, we actually want to check against empty strings here also. So maybe better leave it as is.